### PR TITLE
Home tab - Vincent feedback update

### PR DESCRIPTION
**Pull request by erikfTW**
Vincent feedback update.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -25,12 +23,12 @@ Taipy is a complete tool that empowers individuals and enterprises to develop ap
 **Back-End Functionalities:**
 
 - Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
-- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through data flow functionality.
-- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
-- Facilitating a registry of pipeline executions.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
+- Enhancing the overall performance of Taipy applications, data caching, parallel executions of tasks, data source scoping are all integrated.
+- Providing a registry of pipeline executions.
 - Enabling pipeline versioning.
```
Comment (jrobinAV) [inline docs/index.md]: both lines mean the same, no?

File: docs/index.md [hunk 2]
```diff
@@ -25,12 +23,12 @@ Taipy is a complete tool that empowers individuals and enterprises to develop ap
...
 - Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
-- Furthermore, Taipy offers a built-in visualization feature for pipelines and their associated data.
+- Furthermore, Taipy offers  built-in graphical explorers  for visualizing your ‘executed’ pipelines and their associated data.
```
Comment (jrobinAV) [inline docs/index.md]:
```suggestion
- Furthermore, Taipy offers built-in graphical explorers for visualizing and editing your ‘executed’ pipelines and their associated data.
```

File: docs/index.md [hunk 3]
```diff
@@ -25,12 +23,12 @@ Taipy is a complete tool that empowers individuals and enterprises to develop ap
...
 
 - Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
-- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through data flow functionality.
-- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
-- Facilitating a registry of pipeline executions.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
+- Enhancing the overall performance of Taipy applications, data caching, parallel executions of tasks, data source scoping are all integrated.
+- Providing a registry of pipeline executions.
 - Enabling pipeline versioning.
 - Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
```
Comment (jrobinAV) [inline docs/index.md:29]: I don't know what it is referring to?

File: docs/index.md (hunk 1 above)
Comment (erikfTW) [reply docs/index.md:29]: Well yes in general, let me remove one, thanks.

Comment (erikfTW) [reply docs/index.md]: Yes, in general. Let me remove one, thanks.

File: docs/index.md (hunk 3 above)
Comment (erikfTW) [reply docs/index.md:29]: There is no KPI tracking tool?

File: docs/index.md (hunk 1 above)
Comment (erikfTW) [inline docs/index.md]:
```suggestion
```

File: docs/index.md (hunk 3 above)
Comment (jrobinAV) [reply docs/index.md:29]: Not really. However, we can use the visual elements to monitor some data nodes. I proposed a change in the following line to express that idea.

File: docs/index.md [hunk 4]
```diff
@@ -25,12 +23,11 @@ Taipy is a complete tool that empowers individuals and enterprises to develop ap
...
-- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through data flow functionality.
-- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
-- Facilitating a registry of pipeline executions.
-- Enabling pipeline versioning.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
+- Enhancing the overall performance of Taipy applications, data caching, parallel executions of tasks, data source scoping are all integrated.
+- Providing a registry of pipeline executions.
 - Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
-- Furthermore, Taipy offers a built-in visualization feature for pipelines and their associated data.
+- Furthermore, Taipy offers built-in graphical explorers for visualizing and editing your ‘executed’ pipelines and their associated data.
```
Comment (jrobinAV) [inline docs/index.md]:
```suggestion
- Taipy offers built-in graphical explorers for visualizing, editing, and monitoring your ‘executed’ pipelines and their associated data.
```

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**